task1.py

```diff
@@ -0,0 +1,17 @@
+def search_word(x, my_list):
+    for y in my_list:
```

Comment:
try to think of pythonic solution for this task.

Reply:
Sir I had changed the code.Now have a look at it.